sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -4,7 +4,7 @@
 # -------------------------------------
 from collections import namedtuple
 from ._shared import parse_vault_id
-from ._shared._generated.v7_0.models import JsonWebKey as _JsonWebKey
+from ._generated.v7_0.models import JsonWebKey as _JsonWebKey
```

Comment:
Probably doesn't matter functionally, but I'd import from `v7_1` because in the default case that would avoid importing two sets of models.

Reply:
that's a fair point, going to update with that